src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -192,13 +192,13 @@ private int ComputePort(WebApplication app)
             else if (IsPortInUseByBinding(_application, DefaultPort))
             {
                 // Port has been reserved for the app.
-                app.Logger.LogInformation($"Default dashboard port {DefaultPort} has been reserved by the application, choosing random port.");
+                app.Logger.LogInformation("Default dashboard port {DefaultPort} has been reserved by the application, choosing random port.", DefaultPort);
```

Comment:
Unsure if structure logging was intended here - but majority of code-base uses it, so I updated this to match the majority...

Reply:
Yup. Right now there's a mix of two different logging systems at play here, but the stuff using `ILogger`/Serilog should deffo be structured logs.